src/converters/app-converter.ts

```diff
@@ -543,6 +544,18 @@ export class StackConverter extends ArtifactConverter {
                 }, this.processIntrinsics(params));
             }
 
+            case 'Fn::Equals': {
+                return intrinsics.fnEquals.evaluate(this, params);
```

Comment:
Having separate functions for each intrinsic is much more understandable. Love it!